src/app/media-wall/media-wall.module.ts

```diff
@@ -9,6 +9,7 @@ import { LoklakMediaWallRoutingModule } from './media-wall-routing.module';
 import { MediaWallCardComponent } from './media-wall-card/media-wall-card.component';
 import { MediaWallLinkerComponent } from './media-wall-linker/media-wall-linker.component';
 import { MediaWallHeaderComponent } from './media-wall-header/media-wall-header.component';
+import { ToggleFullscreenDirective } from '../shared//full-screen.directive';
```

Comment:
The path is not correct path why double `//` add this to `index.ts` file of shared folder and use directly